packages/examples/examples/typescript/package.json

```diff
@@ -32,6 +32,7 @@
     "@metamask/eslint-config-jest": "^8.0.0",
     "@metamask/eslint-config-nodejs": "^8.0.0",
     "@metamask/eslint-config-typescript": "^8.0.0",
+    "@metamask/snaps-cli": "workspace:*",
```

Comment:
We only have the CLI added as a dependency for some of the examples. We should decide to either have it added for all examples or none 😅